cmake/onnxruntime_unittests.cmake

```diff
@@ -1210,6 +1210,14 @@ endif()
 
 if (NOT onnxruntime_ENABLE_TRAINING_TORCH_INTEROP)
   if(NOT IOS)
+    onnxruntime_fetchcontent_declare(
+      cxxopts
```

Comment:
>  cxxopts [](http://example.com/codeflow?start=4&length=9)

Does this require new component registration?
Is it available via vcpkg?

Reply:
`cxxopts` is available via vcpkg, but i think ORT relies on `onnxruntime_fetchcontent_declare`/`FetchContent_Declare` and `onnxruntime_fetchcontent_makeavailable`/`FetchContent_MakeAvailable`  to get the 3rd party dependency.

Hmm, from the cmake, we don't build onnxruntime_perf_test for IOS, am i right?

Reply:
> Hmm, from the cmake, we don't build onnxruntime_perf_test for IOS, am i right?

right.

https://github.com/microsoft/onnxruntime/blob/e0ad8050d04c55318d1336874cc6d640509ddbc0/cmake/onnxruntime_unittests.cmake#L1211-L1232

Reply:
Why are we not using the existing abseil functionality and instead introducing new one and two different chunks of code?

Reply:
Updated to use abseil.

Reply:
Thanks!